statistics-dashboard-rest/src/main/java/statistics/dashboard/rest/StatisticsController.java

```diff
@@ -0,0 +1,63 @@
+package statistics.dashboard.rest;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import statistics.dashboard.common.models.StatisticsFilteringRequest;
+import statistics.dashboard.common.view.FilteringResult;
+import statistics.dashboard.common.view.FiltersWrapper;
+import statistics.dashboard.common.view.ResultListFilters;
+
+/**
+ * Controller for the Statistics Dashboard
+ */
+
+@Api("/")
+@Controller
+public class StatisticsController {
```

Comment:
You could use the construction below to make a swagger description for the whole controller.
```
@Tags(@Tag(name = StatisticsController.CONTROLLER_TAG_NAME, description = "..."))
@Api(tags = StatisticsController.CONTROLLER_TAG_NAME)
```

Reply:
Just added them